.prettierrc.js

```diff
@@ -1,15 +1,18 @@
 /* eslint-disable import/no-commonjs */
 module.exports = {
-    overrides: [
-      {
-        files: '*.md',
-        options: {
-          parser: 'markdown',
-          printWidth: 80,
-          proseWrap: 'always',
-          singleQuote: true,
-          trailingComma: 'es5',
-        },
+  overrides: [
```

Comment:
Do we need this override? The only different value is `proseWrap`, which could be relied upon by the editors settings ("View > Toggle word wrap" in VSCode).

Reply:
IMHO the configuration should work with the IDE and the prettier CLI so yes I think we should still have it from the configuration.
As for overriding or not, I actually do not know, I only kept the configuration which was already in place. I will have a look.

Reply:
If we want to keep `proseWrap`, we can get rid of the `overrides` key and set the top-level `proseWrap` value to `"always"` then.

Reply:
Make sense. I have updated the codebase and the configuration accordingly. Please review again